Examples/Python/RunFemur.py

```diff
@@ -0,0 +1,574 @@
+# -*- coding: utf-8 -*-
+"""
+====================================================================
+Full Example Pipeline for Statistical Shape Modeling with ShapeWorks
+====================================================================
+Jadie Adams
+
+The femur data set is comprised of segmented meshes of femurs and corresponding CT images that are not segmented.
+The first step in grooming is to turn the meshes into the binary volume format shapeworks expects.
+The full mages and segmentations must be carried through every stop of grooming. 
+Optimization uses single scale.
+
+
+First import the necessary modules
+"""
+
+from zipfile import ZipFile
+import os
+import sys
+import csv
+import argparse
+import glob
+
+from GroomUtils import *
+from OptimizeUtils import *
+from AnalyzeUtils import *
+
+from GirderConnector import downloadUseCaseData
```

Comment:
I believe this has been replaced with:

from DatasetUtils import datasets
datasets.downloadDataset(filename)

I get an error looking for GirderConnector.  I am assuming it works for you because have the older one installed with pip.

Reply:
Hi Alan, yes this should be replaced. I apologize I've had the zip file downloaded for a while so I didn't realize the Girder code had been changed.

Reply:
Yep, no problem.  I've committed a fix.